### ISSUE TITLE
read_csv OptionError: 'Pattern matched multiple keys' 

### ISSUE THREAD

**Issue by bdklahn** [modin-project/modin#4157]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
https://github.com/NSSAC/ICANS_EIM/pkgs/container/icans-eim-workenv/14448205?tag=0.1.2
(OpenSUSE Tumblweed container)
- **Modin version** (`modin.__version__`): '0.13.0'
- **Python version**: Python 3.8.12
- **Code we can use to reproduce**:

```
#In Jupyter Lab (Version 3.2.9; also available in that container) notebook . . .
navicell_ACSN2_bin_rel_prot_url='https://acsn.curie.fr/ACSN2/downloads/ACSN2_binary_relations_between_proteins_with_PMID.txt'

# This works (pandas version 1.4.0):
navicell_ACSN2_bin_rel_pro_df = pandas.read_csv(navicell_ACSN2_bin_rel_prot_url, sep='\t')
navicell_ACSN2_bin_rel_pro_df

# This is broke:
navicell_ACSN2_bin_rel_pro_mdf = modin.pandas.read_csv(navicell_ACSN2_bin_rel_prot_url, sep='\t')
navicell_ACSN2_bin_rel_pro_mdf

#This is how I installed modin in the container:
"modin[all]>=0.13.0" in pip requirements file
```

### Describe the problem

modin.pandas.read_csv gives error.
pandas.read_csv doesn't. 

![Screenshot_20220204_095126](https://user-images.githubusercontent.com/920400/152549716-6bfdc8a0-a470-430d-b5f9-7a585df6cba9.png)

**Comment by bdklahn** [comment]
Maybe it should now be . . .
`pandas.get_option("display.max_rows")`
?

**Comment by bdklahn** [comment]
Ah . . .
I see this has already been fixed on master . . .
https://github.com/modin-project/modin/pull/4109
I guess I'll wait for the new Pypi package to be pushed (or install from a repo tag/commit).

**Comment by devin-petersohn** [comment]
@bdklahn we have released 0.13.1 recently, which includes the fix for this issue. Please let us know how it goes!

**Comment by bdklahn** [comment]
That worked.
Thanks!
